forge.config.js

```diff
@@ -18,7 +18,7 @@ require('dotenv').config()
 const CONFIG = {
 	// General
 	applicationName: packageJSON.productName,
-	applicationNameSanitized: packageJSON.productName.replaceAll(' ', '-'),
+	applicationNameSanitized: packageJSON.productName.replaceAll(' ', '.'),
```

Comment:
dash is a problem? Would prefer that

Reply:
No. But I didn't like that space replacer and name parts separator are the same symbol 🐧